modules/oxide-components/visual-regression-test-ci.mjs

```diff
@@ -24,13 +24,28 @@ const waitForServer = async () => {
 
 const runTests = async () => {
   try {
-    devServer = exec('yarn start --ci' );
+    devServer = exec('yarn start --ci');
     await waitForServer();
-    execSync('yarn playwright test', {
-      stdio: 'inherit'
-    });
+
+    try {
```

Comment:
This part should be fixed in the Jenkins file so that it doesn't stop execution if a error is returned. I will re review when this is all changed